app/build.gradle

```diff
@@ -24,7 +24,10 @@ dependencies {
     androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    compile 'com.android.support:appcompat-v7:25.3.1'
+    compile 'com.android.support:appcompat-v7:26.0.0-alpha1'
+    compile 'com.android.support:support-v4:26.0.0-alpha1'
+    compile 'com.android.support:recyclerview-v7:+'
+    compile 'com.android.support:cardview-v7:+'
```

Comment:
pozor na pouzivanie `+` v importovani knihoven. moze to narobit problemy v produkcnom kode. je lepsie specifikovat konkretnu verziu knihovny a casom updatovat a testovat ci update nesposobil chyby.